### ISSUE TITLE
Logging happens when set to log level none

### ISSUE THREAD

**Issue by Ankish** [aws-amplify/amplify-swift#3727]
### Describe the bug

No logging should be printed on console when plugin is not added or level is set to none.

Example:
```
 amplifyConfigurationFileName amplifyconfiguration_capture
2024-05-29 14:18:11.114928+0530X[718:286269] [Amplify] Configuring
2024-05-29 14:18:11.121418+0530 X[718:286269] [Amplify] Configuration: Optional(Amplify.AmplifyConfiguration(analytics: nil, api:
2024-05-29 14:18:11.131943+0530 X[718:286263] [AuthenticationAWSCognitoAuthPlugin] Auth state change:
```

### Steps To Reproduce

```swift
If you dont add logging pluging or set to false, still some logging keeps happening exposing URLs etc.

        Amplify.Logging.logLevel = .none

       // let logging = AWSUnifiedLoggingPlugin()
      //  try Amplify.add(plugin: logging)

With this too logging happens with above example messages
```

### Expected behavior

Logging should not happen

### Amplify Framework Version

2.34.1

### Amplify Categories

Analytics, API, Auth, DataStore, Storage

### Dependency manager

Swift PM

### Swift version

5.10

### CLI version

aws-amplify-cli/2.0

### Xcode version

15.3 

### Relevant log output

```shell
<details>
<summary>Log Messages</summary>


Optional(Amplify.AmplifyConfiguration(analytics: nil, api: Optional(Amplify.APICategoryConfiguration(plugins: ["awsAPIPlugin": Amplify.JSONValue.object(["unauthAPI": Amplify.JSONValue.object(["authorizationType": Amplify.JSONValue.string("AWS_IAM"), "region": Amplify.JSONValue.string("us-west-2"), "endpoint": Amplify.JSONValue.string("https://api.up.ai/train_v1"), "endpointType": Amplify.JSONValue.string("REST")]), "production": Amplify.JSONValue.object(["authorizationType": Amplify.JSONValue.string("OPENID_CONNECT"),
```
</details>
```


### Is this a regression?

Yes

### Regression additional context

Logging should not happen

### Platforms

iOS

### OS Version

iOS 16 & iOS 17

### Device

iPhone 13

### Specific to simulators

_No response_

### Additional context

_No response_

**Comment by 5d** [comment]
Thanks for submitting the issue. We will investigate further and provide updates here.

**Comment by 5d** [comment]
The fix has been released in version [2.34.3](https://github.com/aws-amplify/amplify-swift/releases/tag/2.34.3). Please give it a try, and feel free to reopen the ticket if the issue persists.
